backend/tests/test_auth_api.py

```
@@ -25,3 +27,135 @@
+    response = client.delete("/users/9999")
+    assert response.status_code == 404
+    assert response.json()["detail"] == "User not found"
+
+
+
+def test_get_all_products(client):
+    response = client.get("/products")
+    assert response.status_code == 200
+    assert isinstance(response.json(), list)
+
+def test_update_product(client, db_session):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_update_product`

Add return type annotation: `None`

(ANN201)